web/apps/atelier-des-idees/src/components/IdeaCard/index.jsx

```diff
@@ -12,6 +12,23 @@ const AUTHOR_CATEGORY_NAMES = {
     QG: 'LaREM',
 };
 
+const VOTES_NAMES = {
+    important: 'Important',
+    feasible: 'Réalisable',
+    innovative: 'Novateur',
+};
+
+function formatVotes(votesCount) {
+    return Object.keys(votesCount)
+        .filter(key => ['important', 'feasible', 'innovative'].includes(key))
```

Comment:
A quoi sert ce filter ?

Reply:
Pour avoir une unique source de vérité pour les clés de vote
```suggestion
        .filter(key => Object.keys(VOTES_NAMES).includes(key))
```